eng/Versions.props

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <PropertyGroup>
+    <!-- This repo version -->
+    <MajorVersion>3</MajorVersion>
```

Comment:
Starting at version 3.0 seems a bit odd - is this something we need to do for ref package versioning?

Reply:
I was matching it to Release 3.0, but I see your point.  I can change it to 1.0.